payloads/assignpublicIP.go

```diff
@@ -26,37 +26,43 @@ type PublicIPCommand struct {
 	VnicMAC          string `yaml:"vnic_mac"`
 }
 
-// CommandAssignPublicIP is reserved for future use.
+// CommandAssignPublicIP wrapper of PublicIPCommand used during assigment of IP.
 type CommandAssignPublicIP struct {
 	AssignIP PublicIPCommand `yaml:"assign_public_ip"`
 }
 
-// CommandReleasePublicIP is reserved for future use.
+// CommandReleasePublicIP wrapper of PublicIPCommand used during release of IP.
 type CommandReleasePublicIP struct {
 	ReleaseIP PublicIPCommand `yaml:"release_public_ip"`
 }
 
-// PublicIPFailureReason is reserved for future use.
+// PublicIPFailureReason data type to define reasons of failure during Public IP
+// delivery process.
 type PublicIPFailureReason string
 
 const (
-	// PublicIPNoInstance is reserved for future use.
+	// PublicIPNoInstance constant is used to denote when no valid instance exists.
```

Comment:
`PublicIPNoInstance indicates that a public IP assignment/release failed because the corresponding instance does not exist`